modules/ROOT/pages/_partials/attributes.adoc

```diff
@@ -2,7 +2,8 @@
 
 :product-name: Mobile Services
 
-:release-number: 1.0.0
+:release-number: 2.0.0
```

Comment:
@laurafitzgerald  proposed change to 2.0 ;)

Reply:
this comment sounds like I proposed it :P The version bump is only for the SDK so from your comment above i believe this release version relates to the service not the SDK. correct?

Reply:
@laurafitzgerald sorry, I did not mean you suggested it, but now that it has been suggested ;)

@ziccardi  AFAIK we're still at 1.0.0 wrt to this attribute (it's used throughout doc to represent the whole Mobile Services release)